src/vs/base/parts/quickinput/common/quickInput.ts

```diff
@@ -340,6 +340,15 @@ export interface IQuickPick<T extends IQuickPickItem> extends IQuickInput {
 	hideInput: boolean;
 
 	hideCheckAll: boolean;
+
+	/**
+	 * A set of `Toggle` objects to add to the input box.
+	 */
+	additionalToggles: IQuickInputToggle[] | undefined;
```

Comment:
I'm curious why you chose `additionalToggles` instead of just `toggles`. "Additional" implies that there are existing toggles that we are adding on to.

Reply:
Good point, the additional was to align with find input but it doesn't make sense here